bottomless/src/replicator.rs

```diff
@@ -127,6 +151,15 @@ impl Options {
                 options.max_batch_interval = Duration::from_secs(seconds);
             }
         }
+        if let Ok(access_key_id) = std::env::var("LIBSQL_BOTTOMLESS_AWS_ACCESS_KEY_ID") {
+            options.access_key_id = Some(access_key_id);
+        }
+        if let Ok(secret_access_key) = std::env::var("LIBSQL_BOTTOMLESS_AWS_SECRET_ACCESS_KEY") {
+            options.secret_access_key = Some(secret_access_key);
+        }
+        if let Ok(region) = std::env::var("LIBSQL_BOTTOMLESS_AWS_DEFAULT_REGION") {
+            options.region = Some(region);
+        }
```

Comment:
Why not just:
```
options.access_key_id = Some(std::env::var("LIBSQL_BOTTOMLESS_AWS_ACCESS_KEY_ID")?);
options. secret_access_key = Some(std::env::var("LIBSQL_BOTTOMLESS_AWS_SECRET_ACCESS_KEY")?);
options.region = Some(std::env::var("LIBSQL_BOTTOMLESS_AWS_DEFAULT_REGION")?);
```

Reply:
Technically it's overriding the default settings, but only if env vars were set. Atm. we don't have any sensible defaults, but this may change.